template.distillery/Makefile.mobile

```diff
@@ -142,23 +142,23 @@ endif
 
 $(APPJS): $(JS_PREFIX).js
 ifeq ($(APP_REMOTE),yes)
-	APPJS_FILE=$$(curl -s -f $(APP_SERVER)|grep -E -o -m 1 '$(PROJECT_NAME)_[a-f0-9]*?\.js') &&\
+	APPJS_FILE=$$(curl -s -f $(APP_SERVER) | cat | grep -E -o -m 1 '$(PROJECT_NAME)_[a-f0-9]*?\.js') &&\
```

Comment:
Haha! It's because when grep dies, it sends a SIGPIPE to curl, which doesn't like it at all. Like this, it seems to work but I agree it's a bit weird.

Instead of "cat before grep", we could use "head after grep" (as in use grep with `-m 1` and pipe to `head -n 1`) but I'm not sure it'd be much better.